tests/server-snap/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Check snap web servers
+environment:
+  SNAP_NAME/pythonServer: xkcd-webserver
+  IP_VERSION/pythonServer: 4
+  PORT/pythonServer: 80
+  TEXT/pythonServer: XKCD rocks!
+  SNAP_NAME/goServer: go-example-webserver
+  IP_VERSION/goServer: 6
+  PORT/goServer: 8081
+  TEXT/goServer: Hello World
+prepare: |
+  sudo snap install $SNAP_NAME
+  cat > request.txt <<EOF
+  GET / HTTP/1.0
+
+  EOF
+  while ! netstat -lnt | grep -Pq "tcp.*?:$PORT +.*?LISTEN\n*"; do sleep 0.5; done
```

Comment:
Pardon my ignorance on this, will spread automatically kill the test after N minutes if the service does not come up? I assume it does, but maybe a small comment somewhere to assure poor souls like me about it?